### PR TITLE
Submitting March Hearbeat

### PR DESCRIPTION
**Pull request by jendefig**
@shcheklein @jorgeorpinel Here is the March Heartbeat.  I don't like how Elle's video looks from DataTalks.Club.  I've got a screen shot to use, but how to overlay that over this You-tube Video which is showing another presentation?  The video has the talks for a few people so it's long.  Should I see if I can get Alexsey to share with me so I can cut it and put the right graphic on it and then it can also live on our YouTube?  

Also I'm just remembering that I didn't run the pics through tiny.png, so I'll do that now...

### CODE REVIEW

**Comment by jendefig** [comment]
@shcheklein @jorgeorpinel 
I made adjustment to Elle's talk that I'm happy with and other revisions.  Let me know what you see.  This could go out today, if we could get reviewed.

**Comment by jorgeorpinel** [comment]
I don't think `package-lock.json` should be included in these PRs. But shouldn't it be in [.gitignore](https://github.com/iterative/dvc.org/blob/master/.gitignore)? Cc @rogermparent thanks

**Comment by jorgeorpinel** [comment]
@jendefig for now please remove it because it's preventing this PR from deploying (review app) (https://dashboard.heroku.com/apps/dvc-org-march-heartbeat-6vs8b9/activity/builds/dffbb8a5-d929-4986-a958-1fa582faf533)

**Comment by jorgeorpinel** [comment]
Oh wait that's `npm` not `yarn` (technical stufF). Hmmm something funny going on 🙂

File: content/blog/2021-03-15-March-21-dvc-heartbeat.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+commentsUrl: https://discuss.dvc.org/t/february-21-heartbeat/669
+tags:
+  - Heartbeat
+  - CML
+  - DVC
+  - DataTalks.Club
+  - TFIR
+  - Queen's University
+  - arXiv
+  - MLOps
```
Comment (jorgeorpinel) [inline content/blog/2021-03-15-March-21-dvc-heartbeat.md:20]: Haven't reviewed blogs for a while but do we have all these tags? AFAIR we try to keep only a few tags (that already exist in previous blogs) unless it's very relevant to add more.

File: content/blog/2021-03-15-March-21-dvc-heartbeat.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+Welcome to March! It's been a great month already! Here's all that will keep you
+in the know.
+
+![UnderRock](https://media.giphy.com/media/J2gg8fO7RarRgQRC4d/giphy.gif)
+
+## ICYMI - DVC 2.0 is here!
+
+If you somehow missed our March 3rd announcment, DVC 2.0 is here with loads of
+features to make your life easier.
```
Comment (jorgeorpinel) [inline content/blog/2021-03-15-March-21-dvc-heartbeat.md:33]: Link to /blog/dvc-2-0-release ?

**Comment by jendefig** [comment]
@jorgeorpinel @shcheklein Made revisions noted from Jorge (tags and link to blog post), deleted .json file, adjusted pic sizing.

File: content/blog/2021-03-15-March-21-dvc-heartbeat.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
+﻿---
+title: March '21 Heartbeat
+date: 2021-03-15
+description: |
+  Monthly updates are here! News, use cases, videos, journal publications and more!
```
Comment (shcheklein) [inline content/blog/2021-03-15-March-21-dvc-heartbeat.md:5]: please keep it wrapped at 80 symbols, we don't any automation now for this :(. In this case it should something like

```
  Monthly updates are here! News, use cases, videos, journal publications and
  more!
```

probably

File: content/blog/2021-03-15-March-21-dvc-heartbeat.md [hunk 4]
```diff
@@ -0,0 +1,196 @@
+﻿---
+title: March '21 Heartbeat
+date: 2021-03-15
+description: |
+  Monthly updates are here! News, use cases, videos, journal publications and
+  more!
+descriptionLong: |
+  Monthly updates are here! Read all about our growing team, 
+  our CEO's interview on TFIR, Elle's talk at DataTalks.Club Conference and more!
```
Comment (shcheklein) [inline content/blog/2021-03-15-March-21-dvc-heartbeat.md:9]: 💅 same here - wrap at 80 please

File: content/blog/2021-03-15-March-21-dvc-heartbeat.md [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+title: March '21 Heartbeat
+date: 2021-03-15
+description: |
+  Monthly updates are here! News, use cases, videos, journal publications and more!
+descriptionLong: |
+  Monthly updates are here! Read all about our growing team, 
+  our CEO's interview on TFIR, Elle's talk at DataTalks.Club Conference and more!
+picture: 2021-03-15/mar21cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/february-21-heartbeat/669
```
Comment (shcheklein) [inline content/blog/2021-03-15-March-21-dvc-heartbeat.md:12]: reminder: update the link before merge

**Review by shcheklein — APPROVED** [review]
Besides some minor style thingies and updating the link, all looks good to me! Really great stuff, let's ship it.

**Comment by jendefig** [comment]
@shcheklein is the 80 character limit true for links too?  I did all regular text that was too long but not the links

**Review by shcheklein — APPROVED** [review]

**Pull request merged**